automation_tools/__init__.py

```diff
@@ -1140,13 +1154,15 @@ def client_registration_test(clean_beaker=True, update_packages=True):
     if not cert_url:
         print('You need to install the Candlepin Cert RPM.')
         sys.exit(1)
-
+    # Release version for the client - Optional - no need to error out if not
+    # available
+    rel_ver = os.getenv('RELVER')
```

Comment:
Should this task fail if that env var is not defined? If yes, then I recommend that you check at the beginning of the task before doing anything on the server and fail if something is not defined.